COVID_19_model.ipynb

```diff
@@ -1404,11 +1405,14 @@
         "def plot_country_posterior(\n",
         "  plot_title,\n",
         "  predict_data,\n",
+        "  simple_model_data,\n",
```

Comment:
I noticed that the plot function didn't need this arg, so I dropped it. You might want to do the same.

Reply:
Dropped